README.md

```diff
@@ -1,6 +1,6 @@
 # `tatm` Kempner AI Testbed Library
 
-`tatm` (**T**ransformer **A**ssistive **T**estbed **M**odule) is a Python library that provides a set of tools to assist in the development of transformer-based (and other architecture) models for research within the Kempner institute. It will provide an interface for accessing and manipulating data, training models, and evaluating models. The library is designed to be modular and extensible, allowing for easy integration of new models, datasets, and training frameworks.
+`tatm` (**T**ransformer **A**ssistive **T**estbed **M**odule) is a Python library that provides a set of tools to assist in the development of transformer-based (and other architecture) models for foundation model research on on prem clusters. It currently provides an interface for accessing and manipulating data, and in the future will include features for training models and evaluating models. The library is designed to be modular and extensible, allowing for easy integration of new models, datasets, and training frameworks.
```

Comment:
Can you replace "on prem" with "on-premise"? Looks better given the "on" right before it.

Reply:
done